### DISCUSSION TITLE
Powerful editor on poor-quality scores

### DISCUSSION THREAD

**Discussion by hbitteur** [Audiveris/audiveris#491] [General]
* Focus on poor-quality scores
* Detection of tablatures
* Support for 1-line percussion staves, for merged grand staff
* Support for cross heads, implicit tuplets, non measure-long whole rests
* Ability to manually edit any Inter instance
* Ability to insert compound notes, key signature cancel
* Ability to guide voice and time slot for chords
* Ability to limit book processing on selected sheets
* Handbook hosted on GitHub pages

<hr /><em>This discussion was created from the release <a href='https://github.com/Audiveris/audiveris/releases/tag/v5.2.1'>Powerful editor on poor-quality scores</a>.</em>

**Comment by franckmusescore** [comment]
Hello Hervé
I like the new possibilities to correct inter, but it seems to do worse on some issues  :  many errors on this IMSLP file
[1_IMSLP25308-PMLP03845-Mozart_Figaro_K.492_Act_1_I--IV.pdf](https://github.com/Audiveris/audiveris/files/6743115/1_IMSLP25308-PMLP03845-Mozart_Figaro_K.492_Act_1_I--IV.pdf)

Please have a look. 
Most of my work published on Musescore.com  started with Audiveris
https://musescore.com/user/11173771

I will be happy to make more tests if you like.

Regards
Franck

**Comment by hbitteur** [comment]
Hi Franck,
Could you pull Audiveris from github and checkout the development branch?
I recently fixed recognition problems that Baruch had reported (see issue #498)
I will check your pdf file, but I'm really busy right now.
Bye,
/Hervé

**Reply by franckmusescore** [reply]
hello Hervé
not quite sure on how to do that. I tried open pull request in the dev branch...

Franck

**Comment by hbitteur** [comment]
My advice was for you to rebuild Audiveris application from the latest development version.
This implies you clone Audiveris project from github to your local machine, checkout the development branch and then launch the build command.

Assuming you are not familiar enough with this approach, I will use your input PDF file on my local development version and check the result. I'll keep you informed.

FYI, I have a few potential bugs to investigate before I publish the coming 5.2.2 release (for you, this will be more convenient to use than playing with the building of development material).

**Reply by franckmusescore** [reply]
Hello Hervé,
thanks for your kind reply.
I managed to install Git Desktop, clone development branch, and build todays version. (with your work and slur ties)
 
Bonsoir,
Franck
